dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/DatasetPage.java

```diff
@@ -4306,7 +4306,8 @@ public boolean isThisLatestReleasedVersion() {
     public String getJsonLd() {
         if (isThisLatestReleasedVersion()) {
             ExportService instance = ExportService.getInstance(settingsService);
-            String jsonLd = instance.getExportAsString(dataset, SchemaDotOrgExporter.NAME);
+            String jsonLd = instance.getExportAsString(dataset, SchemaDotOrgExporter.NAME,
+                    settingsService.isTrueForKey(SettingsServiceBean.Key.ExcludeEmailFromExport));
```

Comment:
I think that we should not change ExportService method definitions but to use ExportService.settingsService